src/components/TodoList.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Todo } from './Todo';
+import './TodoList.css';
+
+export const TodoList = ({ preparedTodos }) => (
+  <>
+    <tr>
+      <th>Todos</th>
+      <th>Is completed</th>
+      <th>Users</th>
+    </tr>
+    {preparedTodos.map(todo => (
+      <tr className="todo" key={todo.id}>
+        <Todo todo={todo} />
```

Comment:
```suggestion
        <Todo todoData={singleTodo} />
```
три слова todo подряд это слишком много